### PR TITLE
fix accessbility issue

### PR DESCRIPTION
**Pull request by yuzdeng**
fix: bugfix

Detail:
To fix the accessibility issues of pagination. Now the inputbox and select of pagination have the accessibility issues. For the detail please see the issues: https://github.com/react-component/pagination/issues/345

Solution:
Add the `aria-label` and `defaultOpen ` value for the select and input component.

### CODE REVIEW

File: src/Options.jsx [hunk 1]
```diff
@@ -123,6 +123,8 @@ class Options extends React.Component {
           value={(pageSize || pageSizeOptions[0]).toString()}
           onChange={this.changeSize}
           getPopupContainer={(triggerNode) => triggerNode.parentNode}
+          aria-label={locale.page_size}
+          defaultOpen={false}
```
Comment (afc163) [inline src/Options.jsx:127]:
```suggestion
```

`defaultOpen={false}` could be deleted.

Comment (yuzdeng) [reply src/Options.jsx:127]: We need defaultOpen to set the `Input` components accessibility issue. For the detail you can check my this comment: https://github.com/ant-design/ant-design/issues/31692#issuecomment-914906885

Comment (yuzdeng) [reply src/Options.jsx:127]: @afc163 Any other problems here?

**Comment by manikandanm1757** [comment]
Can anybody update in which version this fix available?

**Comment by yoyo837** [comment]
> Can anybody update in which version this fix available?

image

**Pull request merged**